### ISSUE TITLE
App loading screen displayed forever when creating a new Status account

### ISSUE THREAD

**Issue by saledjenic** [status-im/status-desktop#11454]
Cannot create a new Status account neither log in an existing one cause something is broken and loading screen is displayed forever.

**Comment by jrainville** [comment]
For some reason, @iurimatias is getting this in a 0.13 build. Is is the same error you fixed @saledjenic ?

```
INF 2023-07-10 10:15:22.702-04:00 starting application controller...         topics="status-app" tid=8548093 file=nim_status_client.nim:186
Debug: Network Cache Dir:  "/Users/iurimatias/Library/Application Support/Status/tmp/netcache" (:0, )
INF 2023-07-10 10:15:23.330-04:00 starting application...                    topics="status-app" tid=8548093 file=nim_status_client.nim:189
WRN 2023-07-10 10:15:23.333-04:00 Error decoding signal                      topics="signals-manager" tid=8548093 file=signals_manager.nim:48 err="Unknown signal received: mediaserver.started"
Warning: Populating font family aliases took 99 ms. Replace uses of missing font family "Inter" with one that exists to avoid this cost.  (:0, )
ERROR[07-10|10:15:26.189] failed to initialize db                  package=status-go/api.GethStatusBackend err="failed to set `journal_mode` pragma"
failed to set `journal_mode` pragma
ERR 2023-07-10 10:15:26.189-04:00 error:                                     topics="accounts-service" tid=8548093 file=service.nim:602 procName=verifyDatabasePassword errDesription="failed to set `journal_mode` pragma"
WRN 2023-07-10 10:15:26.261-04:00 Error decoding signal                      topics="signals-manager" tid=8548093 file=signals_manager.nim:48 err="Unknown signal received: mediaserver.started"
```

**Comment by saledjenic** [comment]
@jrainville yes, looks like it is.

**Comment by jrainville** [comment]
weird that it happened to him on 0.13. Would that fix help? Might be worth cherry-pick in a branch and see if it helps.

**Comment by saledjenic** [comment]
@jrainville I guess that cherry pick will help. 
This issue handles changes made in status-go commit `e08aad7d731817169d6c7140fabc7e02d277dd19`, further means if release build anyhow refers to status-go which includes that commit, then PR handling this issue must help.

**Comment by saledjenic** [comment]
I've just checked, and yes, that commit I mentioned about is not part of the status-go targeted by release build.

> Maybe the issue is not just part of that commit?

Changes done here are definitely desktop app handling for the commit `e08aad7d731817169d6c7140fabc7e02d277dd19`

But it could be there is something more which blocks a user to proceed. Since it's not this, it looks to me that it's something I logged here: https://github.com/status-im/status-desktop/issues/11446 but I haven't investigated that yet.
